app/components/PageGrid.tsx

```diff
@@ -76,6 +76,10 @@ export function PageGrid({ left, subtitle, title }: PageGridProps) {
 						rotate={-45}
 					/>
 				</a>
+				<span className="page-grid-footer-separator" />
+				<a className="page-grid-footer-link" href="/sponsors">
```

Comment:
This line is 45px tall, while the rest are ~55px because of the icon, causing this one to feel a little more scrunched into the others, I think!

Reply:
I added a link to the LinkedIn group, which resolves this issue and balances the footer links.